src/dotnet-uninstall/Shared/BundleInfo/BundleArch.cs

```diff
@@ -1,5 +1,8 @@
-﻿namespace Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo
+﻿using System;
+
+namespace Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo
 {
+    [Flags]
```

Comment:
haven't use this myself. maybe talk to Nick for the best practice? But that's fine to checkin since we already have good test coverage